README.md

```diff
@@ -9,6 +9,7 @@
 
 This document explains the Order Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
 This extension allows assets ordering management within STAC specification.
+In the context of this extension, the term "order" refers to a request for data to be delivered, served, or otherwise made available (e.g., after a purchase).
```

Comment:
```suggestion
In the context of this extension, the term "order" refers to a request for data to be
delivered, served, or otherwise made available (e.g., after a purchase).
```